### ISSUE TITLE
How to deal with VCard? 

### ISSUE THREAD

**Issue by CiaranWelsh** [sys-bio/libOmexMeta#50]
Here is a snippet from a biomodels xml: 

```
                <rdf:Description rdf:about="#_272044">
                    <dc:creator>
                        <rdf:Bag>
                            <rdf:li rdf:parseType="Resource">
                                <vCard:N rdf:parseType="Resource">
                                    <vCard:Family>Chelliah</vCard:Family>
                                    <vCard:Given>Vijayalakshmi</vCard:Given>
                                </vCard:N>
                                <vCard:EMAIL>viji@ebi.ac.uk</vCard:EMAIL>
                                <vCard:ORG rdf:parseType="Resource">
                                    <vCard:Orgname>EMBL-EBI</vCard:Orgname>
                                </vCard:ORG>
                            </rdf:li>

                            <rdf:li rdf:parseType="Resource">
                                <vCard:N rdf:parseType="Resource">
                                    <vCard:Family>Nikoloski</vCard:Family>
                                    <vCard:Given>Zoran</vCard:Given>
                                </vCard:N>
                                <vCard:EMAIL>nikoloski@mpimp-golm.mpg.de</vCard:EMAIL>
                                <vCard:ORG rdf:parseType="Resource">
                                    <vCard:Orgname>Institute of Biochemistry and Biology, University of Potsdam, 14476
                                        Potsdam, Germany
                                    </vCard:Orgname>
                                </vCard:ORG>
                            </rdf:li>
                            <rdf:li rdf:parseType="Resource">
                                <vCard:N rdf:parseType="Resource">
                                    <vCard:Family>Arnold</vCard:Family>
                                    <vCard:Given>Anne</vCard:Given>
                                </vCard:N>
                                <vCard:EMAIL>arnold@mpimp-golm.mpg.de</vCard:EMAIL>
                                <vCard:ORG rdf:parseType="Resource">
                                    <vCard:Orgname>Max-Planck-Institute of Molecular Plant Physiology</vCard:Orgname>
                                </vCard:ORG>
                            </rdf:li>
                        </rdf:Bag>
                    </dc:creator>
        ...........................................
```

This needs to be translated somehow. What should the resulting triples look like after translating?

**Comment by CiaranWelsh** [comment]
This issue will be dealt together with #59 because they are related. However, this issue cannot progress further until there is an appropriate translation for VCard.

**Comment by jhgennari** [comment]
Okay, here's a stab at a translation of the VCard information in the foaf standard: 

    vCard:Email --> foaf:mbox
    vCard:Family --> foaf:familyName
    vCard:Given --> foaf:givenName
    vCard:Orgname --> foaf:Organization

We should also get rid of the rdf:Bag construct (see issue #49 ). Also the construct "vCard:N" is unnecessary and should be dropped. (This information is captured in the foaf terms familyName and givenName.) Likewise for vCard:ORG. 

In the example above, you've got three names listed as "dc:creator" for this model. As discussed in email, we are going to assume that these folk are all curators (rather than model authors). Thus, you'd need three lines as follows:

<http://omex-library.org/ModelName.OMEX/ModelName.rdf>  dc:creator  local:entity123 . 
<http://omex-library.org/ModelName.OMEX/ModelName.rdf>  dc:creator  local:entity124 . 
<http://omex-library.org/ModelName.OMEX/ModelName.rdf>  dc:creator  local:entity125 . 

And then, for example, the first curator would look like: 

local:entity123 foaf:familyName "Chelliah" ;
                        foaf:givenName "Vijayalakshmi"; 
                        foaf:organization "EMBL-EBI"; 
                        foaf:mbox "viji@ebi.ac.uk" . 

Note that the last triple could also be foaf:mbox <mailto:viji@ebi.ac.uk> 

Then you would need similar local entities for Zoran N and Anne A, the other two curators. 

Hope that's clear!
-John G.

**Comment by CiaranWelsh** [comment]
resolved with #62
